opencga-catalog/src/main/java/org/opencb/opencga/catalog/db/mongodb/iterators/OrganizationCatalogMongoDBIterator.java

```diff
@@ -78,6 +89,20 @@ private void fetchNextBatch() {
                 internal.put("migrationExecutions", migrationRuns);
             }
 
+            if (includeField(options, OrganizationDBAdaptor.QueryParams.PROJECTS.key())) {
+                OpenCGAResult<Document> openCGAResult = null;
+                try {
+                    if (StringUtils.isNotEmpty(user)) {
+                        openCGAResult = projectMongoDBAdaptor.nativeGet(clientSession, new Query(), projectOptions, user);
+                    } else {
+                        openCGAResult = projectMongoDBAdaptor.nativeGet(clientSession, new Query(), projectOptions);
+                    }
+                } catch (CatalogDBException | CatalogAuthorizationException e) {
+                    logger.warn("Could not fetch projects for organization.", e);
```

Comment:
Why are you ignoring every catalog exception here? I guess that you should only ignore the "authorization" exceptions?

Reply:
Because I don't see why I should be failing to be honest. If the user is attempting to fetch the Organization document and everything is OK, I can't think of any reason why I should fail in case I can't fetch the project information to make the Organization info more complete. I may be wrong and in that case, I would need to fix not only this OrganizationIterator but all of them because I always follow the same approach.

Reply:
But what if the error is a wrapped MongoDB timeout error, or a wrapped NPE exception? The only type of exceptions that make kind of sense that you ignore here is are the authorization exceptions, nothing else